pkg/apis/apps/v1alpha1/uniteddeployment_types.go

```diff
@@ -169,6 +169,9 @@ type UnitedDeploymentCondition struct {
 	// Status of the condition, one of True, False, Unknown.
 	Status corev1.ConditionStatus `json:"status,omitempty"`
 
+	// The last time this condition was updated.
+	LastUpdateTime metav1.Time `json:"lastUpdateTime,omitempty"`
```

Comment:
Is this field updated anywhere?

Reply:
Removed this field and added the condition supports.